smriprep/workflows/surfaces.py

```diff
@@ -474,6 +482,9 @@ def init_gifti_surface_wf(*, name='gifti_surface_wf'):
         (surface_list, fs2gii, [('out', 'in_file')]),
         (fs2gii, fix_surfs, [('converted', 'in_file')]),
         (inputnode, fix_surfs, [('fsnative2t1w_xfm', 'transform_file')]),
+        (fix_surfs, euler, [('out_file', 'in_file')]),
```

Comment:
I believe we just need to calculate the Euler number on each hemisphere of the wm surface. We could store both in the metadata of the `_desc-preproc_T1w.nii.gz` image.